Wabbajack/UI/ReactiveMetro.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Windows;
+using MahApps.Metro.Controls;
+using ReactiveUI;
+
+namespace Wabbajack.UI
+{
+    public class ReactiveMetro<TViewModel> : MetroWindow, IViewFor<TViewModel> where TViewModel : class
+    {
+        public static readonly DependencyProperty ViewModelProperty =
+            DependencyProperty.Register(
+                "ViewModel",
```

Comment:
Might've just been copy/pasted from the RxUI website like this, but I'd recommend `nameof(ViewModel)`